src/status.py

```diff
@@ -7,6 +11,7 @@
 from pvsite_datamodel.read.status import get_latest_status as get_latest_status_site
 from pvsite_datamodel.sqlmodels import StatusSQL
 
+STATUS_API_URL = os.getenv("STATUS_API_URL", "https://status.quartz.energy")
```

Comment:
you can actually use "ENVIRONMENT" which is set to development or production to choose between your different status endpoints

Reply:
https://github.com/openclimatefix/analysis-dashboard/blob/main/src/satellite_page.py#L11